tmt/package_managers/dnf.py

```diff
@@ -26,8 +27,10 @@ class Dnf(tmt.package_managers.PackageManager):
     probe_priority = 50
 
     _base_command = Command('dnf')
+    _base_debuginfo_command = Command('debuginfo-install')
```

Comment:
I wonder whether it wouldn't be better to do `dnf debuginfo-install`, as `debuginfo-install` is just an alias provided by `yum-utils` package. If it's not installed on the system, `dnf debuginfo-install` still works well, but just `debuginfo-install` won't.

Reply:
No idea. I for one am fine with `debuginfo-install`, and I'm happy it finally works, and it's a bug we need to fix, therefore a mere improvement is probably not a good benefit a day before the end of the release window.

I'm open to improvements, we can open new PR right away, for 1.35. I suppose the same goes for `yum`, `yum debuginfo-install` would work. Would we need to perform somealternative to `self.install(FileSystemPath('/usr/bin/debuginfo-install'))`?

Reply:
Thanks for explanation.

Reply:
Filed an issue to track the change, https://github.com/teemtee/tmt/issues/2974